catalogue_graph/src/ingestor_local.py

```diff
@@ -38,7 +36,13 @@ def main() -> None:
         "--pipeline-date",
         type=str,
         help="The date to use for the pipeline, required.",
-        required=True,
+        required=False,
+    )
+    parser.add_argument(
+        "--limit",
+        type=int,
+        help="The number of shards to process, will process all if not specified.",
```

Comment:
Is it clear enough what shard means here, given the context? (parquet shard vs. DB shard) I think it is.

Reply:
I'm not sure, this was a bit of an afterthought to allow this to be tested. I don't expect this will be used outside of testing but agree it's not super clear.